app/src/main/java/com/github/polypoly/app/menu/JoinGameLobbyActivity.kt

```diff
@@ -130,6 +137,9 @@ class JoinGameLobbyActivity : ComponentActivity() {
         var text by remember { mutableStateOf("") }
 
         OutlinedTextField(
+            colors = TextFieldDefaults.outlinedTextFieldColors(
```

Comment:
You should use the defined text field colors `UIElements.outlineTextFieldColors()`